src/panels/calendar/ha-panel-calendar.js

```diff
@@ -0,0 +1,209 @@
+import '@polymer/app-layout/app-header-layout/app-header-layout.js';
+import '@polymer/app-layout/app-header/app-header.js';
+import '@polymer/app-layout/app-toolbar/app-toolbar.js';
+import '@polymer/paper-dropdown-menu/paper-dropdown-menu.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/paper-input/paper-input.js';
+import '@polymer/paper-item/paper-item.js';
+import '@polymer/paper-dialog/paper-dialog.js';
+import '@polymer/paper-listbox/paper-listbox.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import '@vaadin/vaadin-date-picker/vaadin-date-picker.js';
+
+import '../../components/ha-menu-button.js';
+import '../../components/state-history-charts.js';
+import '../../data/ha-state-history-data.js';
+import '../../resources/ha-date-picker-style.js';
+import '../../resources/ha-style.js';
+import './ha-big-calendar.js';
+
+import LocalizeMixin from '../../mixins/localize-mixin.js';
+
+/*
+ * @appliesMixin LocalizeMixin
+ */
+class HaPanelCalendar extends LocalizeMixin(PolymerElement) {
+  static get template() {
+    return html`
+      <style include="iron-flex ha-style">
+        .content {
+          padding: 16px;
+          @apply --layout-horizontal;
+        }
+
+        ha-big-calendar {
+          min-height: 500px;
+          min-width: 100%;
+        }
+
+        #calendars {
+          padding-right: 16px;
+          width: 15%;
+        }
+
+        div.all_calendars {
+    ￼     height: 20px;
+    ￼     text-align: center;
+        }
+
+        span.calendar_color {
+          width: 15px;
+          height: 5px;
+     ￼    margin: 0px 10px 2px 0px;
+     ￼    display: inline-block;
+          background-color: #3174ad;
+        }
+     
+        span.calendar_color_spacer {
+          width: 15px;
+        }
+
+        .iron-selected {
+          background-color: #e5e5e5;
+          font-weight: normal;
+        }
+
+        paper-dialog > div.eventContent {
+          margin: 0px 5px 15px 5px;
+        }
+
+        paper-icon-item {
+          padding: 0px; 
+        }
+
+        paper-item-body.start {
+            padding: 0px 30px 0px 0px;
+        }
+
+        paper-dialog {
+            padding: 0px;
+        }
+
+        paper-dialog > h2 {
+            margin: 0;
+            padding: 20px;
+            background-color: #03A9F4;
+        }
+      </style>
+
+    <app-header-layout has-scrolling-region>
+      <app-header slot="header" fixed>
+        <app-toolbar>
+          <ha-menu-button narrow='[[narrow]]' show-menu='[[showMenu]]'></ha-menu-button>
+          <div main-title>[[localize('panel.calendar')]]</div>
+        </app-toolbar>
+      </app-header>
+
+      <div class="flex content">
+        <div id="calendars" class="layout vertical wrap">
+          <paper-card heading="Calendars">
```

Comment:
For a future PR, we should extract calendar picking into its own component.

Reply:
Do you we I remove this from this PR ?

Reply:
Just ignore this comment for now.